benchmarks/python/core.py

```diff
@@ -23,6 +22,8 @@
 L2_CACHE_SIZE = DEVICE_PROPERTIES["gpu_l2_bytes"]
 PEAK_BANDWIDTH_GBPS = DEVICE_PROPERTIES["gpu_peak_bandwidth_gbps"]
 
+DEFAULT_EXECUTORS = ["eager", "torchcompile", "thunder"]
```

Comment:
Maybe this should be named differently since these are not run in nightly, but for most benchmarks, these are the set of executors we execute weekly. We also have `thunder-torchcompile` for RoPE.
Maybe `BASELINE_EXECUTORS` is better, although Thunder is not really a baseline.

Reply:
speaking of `Thunder is not really a baseline`. I think @naoyam made a comment about renaming `thunder` to `thunder-nvfuser`, which I think makes sense, since we are fixing the executor to only allow nvfuser in those benchmarks.

I'll try to start a PR renaming that. Does that sound reasonable to you?
cc'ing @xwang233 sorry we might need to also update our nightly/weekly scripts.

Reply:
not a problem

Reply:
Yes, I recall that. I can do that in a separate PR so it is strictly a string change. May be easier to review.